docs/source/docs/experimental.rst

```diff
@@ -1,12 +1,10 @@
 Experimental Features
 =====================
 
-There are some experimental features provided in DeepReg,
-which are still being developed but can already be used.
-
-The following tutorials are provided to explain these features.
-Please feel free to `raise an issue`_ to give us feedback.
+DeepReg provides some experimental features.
+These are still in development but can be used.
 
+The following tutorials provide an overview of these features. To submit feedback, open a `new issue <https://github.com/DeepRegNet/DeepReg/issues/new>`__.
```

Comment:
this is rst file not md file, so the grammar is a bit different.

instead of 
```
open a `new issue <https://github.com/DeepRegNet/DeepReg/issues/new>`__.
```

we can write
```
`open a new issue`.
```

with 
```
.. _open a new issue: https://github.com/DeepRegNet/DeepReg/issues/new
```

or simply delete the line 11 raise an issue ;)